examples/README.md

```diff
@@ -77,37 +89,37 @@ with the lane owner.
 Here are the possible obstacles and the action you should take when hitting
 them:
 
```

Comment:
We need a table for the list bellow, it is not readable in the current form.

We also have an issue with the world content, it is not readable as plain text, maybe use code block for it?